pkg/virt-controller/services/vm.go

```diff
@@ -48,8 +48,8 @@ type VMService interface {
 	GetRunningMigrationPods(*corev1.Migration) (*v1.PodList, error)
 	CreateMigrationTargetPod(migration *corev1.Migration, vm *corev1.VM) error
 	UpdateMigration(migration *corev1.Migration) error
-	FetchVM(vmName string) (*corev1.VM, bool, error)
-	FetchMigration(migrationName string) (*corev1.Migration, bool, error)
+	FetchVM(vmName string, namespace string) (*corev1.VM, bool, error)
```

Comment:
Instead of using `FetchVM`, you can start using the `KubevirtClient` from kubecli.go. It is modelled according to the kubernetes core client API.